### PR TITLE
fix: replaced all  edit name with Rename

### PR DESCRIPTION
**Pull request by saiprabhu-dandanayak**
### Description

Fixes : [#34533](https://github.com/appsmithorg/appsmith/issues/34533)

I have raised this PR ` I order to replace all occurances of Edit name with Rename`

### Queries Tab
image

### Js Tab 
image

### Ui Tab
image

## Summary by CodeRabbit

- **Refactor**
  - Updated context menu option from "Edit name" to "Rename" across the app for consistency.

- **Style**
  - Unified naming convention in context menus to improve user interface coherence.

### CODE REVIEW

File: app/client/src/pages/Editor/Explorer/ContextMenu.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default function TreeDropdown(props: TreeDropdownProps) {
   const handleSelect = (option: TreeDropdownOption) => {
     if (option.onSelect) {
       // MenuTrigger takes focus after the Menu closes. For an input to be focused for e.g
-      // edit name we have to take focus back from it.
+      // Rename we have to take focus back from it.
```
[the remark this answers is not in the document]
Comment (saiprabhu-dandanayak) [reply app/client/src/pages/Editor/Explorer/ContextMenu.tsx:54]: resolved

File: app/client/src/ce/constants/messages.ts [hunk 2]
```diff
@@ -1712,7 +1712,7 @@ export const END_DESCRIPTION = () =>
   "Inspect properties of queries, components, etc.";
 export const END_BUTTON_TEXT = () => "Start building an app";
 
-export const CONTEXT_EDIT_NAME = () => "Edit name";
+export const CONTEXT_RE_NAME = () => "Rename";
```
[the remark this answers is not in the document]
Comment (saiprabhu-dandanayak) [reply app/client/src/ce/constants/messages.ts]: resolved

**Comment by saiprabhu-dandanayak** [comment]
Hi **@ApekshaBhosale**  , Could you please review this PR .

**Comment by saiprabhu-dandanayak** [comment]
> Hi **@ApekshaBhosale** , Could you please review this PR .

Hi @ApekshaBhosale ,
I hope you're well. I’m curious and excited to hear your thoughts on my pull request raised a while ago. Looking forward to your review!
Thank you.

**Comment by saiprabhu-dandanayak** [comment]
> This PR has not seen activitiy for a while. It will be closed in 7 days unless further activity is detected.

Hi @Nikhil-Nandagopal , if you have enough bandwidth , could you please review this pr.

**Comment by rohan-arthur** [comment]
@saiprabhu-dandanayak thanks for submitting this!
Automated checks and tests are running here: https://github.com/appsmithorg/appsmith/pull/34871
You can also manually test the changes in this deploy preview: https://ce-34570.dp.appsmith.com/

Meanwhile, we are reviewing this change too.

**Comment by saiprabhu-dandanayak** [comment]
> @saiprabhu-dandanayak thanks for submitting this! Automated checks and tests are running here: #34871 You can also manually test the changes in this deploy preview: https://ce-34570.dp.appsmith.com/
> 
> Meanwhile, we are reviewing this change too.

Hi @rohan-arthur , i have checked the changes in the above mentioned deploy preview , edit name is is replaced with Rename and functionality is working as later one.

File: app/client/src/pages/Editor/Explorer/Files/FilesContextProvider.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React, { createContext, useMemo } from "react";
 import type { ActionParentEntityTypeInterface } from "@appsmith/entities/Engine/actionHelpers";
 
 export enum ActionEntityContextMenuItemsEnum {
-  EDIT_NAME = "Edit Name",
+  RE_NAME = "Rename",
```
Comment (alex-golovanov) [inline app/client/src/pages/Editor/Explorer/Files/FilesContextProvider.tsx:5]: Rename is a single word, pleas change `RE_NAME` to `RENAME`.

Comment (saiprabhu-dandanayak) [reply app/client/src/pages/Editor/Explorer/Files/FilesContextProvider.tsx]: replaced RE_NAME with RENAME

**Review by alex-golovanov — DISMISSED** [review]

**Comment by saiprabhu-dandanayak** [comment]
Hi @alex-golovanov , i have got a merge conflict , will pull the recent changes and update this pr can you approve it again .

**Comment by sagar-qa007** [comment]
@saiprabhu-dandanayak Please run all test again. Otherwise PR looks good to me.

**Comment by saiprabhu-dandanayak** [comment]
> @saiprabhu-dandanayak Please run all test again. Otherwise PR looks good to me.

Hi @sagar-qa007 , i ran the testcases , below is the screenshot
image

**Comment by sagar-qa007** [comment]
@saiprabhu-dandanayak I am asking running the E2E case with this PR as it has frontend changes.

**Comment by saiprabhu-dandanayak** [comment]
> @saiprabhu-dandanayak I am asking running the E2E case with this PR as it has frontend changes.

Hi **@sagar-qa007** , i have run the E2E  cypress test for this pr , below are the generated videos.

https://github.com/user-attachments/assets/64baabca-a010-4153-a8d2-90568e452d40

https://github.com/user-attachments/assets/781dbe32-e1f5-4bec-a054-a824e8d583be

https://github.com/user-attachments/assets/1511f028-d8ef-4bd5-9bae-276f237a7207

File: app/client/cypress/manual_TestSuite/API_Datasource_Spec.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+const commonlocators = require("../../../locators/commonlocators.json");
+
+describe("API associated with Datasource", function () {
```
Comment (sagar-qa007) [inline app/client/cypress/manual_TestSuite/API_Datasource_Spec.js:3]: Please remove this file as we are not maintaining it and deleted.

Comment (saiprabhu-dandanayak) [reply app/client/cypress/manual_TestSuite/API_Datasource_Spec.js]: deleted the file.

**Review by alex-golovanov — APPROVED** [review]

**Comment by saiprabhu-dandanayak** [comment]
Hi **@alex-golovanov** , could you please merge this pr if everything is ok.

**Comment by alex-golovanov** [comment]
> Hi **@alex-golovanov** , could you please merge this pr if everything is ok.

Hey @saiprabhu-dandanayak, looks like there is a stuck test.  I will ask to force merge it.

**Comment by saiprabhu-dandanayak** [comment]
> > Hi **@alex-golovanov** , could you please merge this pr if everything is ok.
> 
> Hey @saiprabhu-dandanayak, looks like there is a stuck test. I will ask to force merge it.

> Hi **@alex-golovanov , @sagar-qa007** , recently start-dev-server.sh file is been deleted through this [commit](https://github.com/appsmithorg/appsmith/pull/34858) and respective server-setup.md file is not updated , could you pls give inputs here

**Pull request merged**